exporters/exporter.go

```diff
@@ -198,6 +198,10 @@ func NewExporter(name, prefix, cloud string, disabledMetrics []string, endpointT
 		transport = &http.Transport{TLSClientConfig: tlsConfig}
 	}
 
+	if transport != nil {
```

Comment:
is this validation required? could be new()'ed before?

Reply:
yes earlier I tried to initialize it above the bock:
```diff
+	transport.Proxy = &http.Transport{http.ProxyFromEnvironment}
	if !*config.Verify {
		log.Infoln("SSL verification disabled on transport")
		tlsConfig := &tls.Config{InsecureSkipVerify: true}
-		transport = &http.Transport{TLSClientConfig: tlsConfig}
+		transport.TLSClientConfig = tlsConfig
	}
```

but didn't know if it would have any consequences for cases where `config.Verify` is `true` or for non SSL cases, because the `transport` would then always be non-nil in the below line 
```
client, err := NewServiceClient(name, &opts, transport, endpointType)
```